components/avatar/avatar.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- font awesome CDN link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="avatar.css">
+    <title>Falcon UI | Home</title>
+</head>
+<body>
+    <div class="main-container">
+        <h1 class="mgn">Avatar</h1>
+        <p>An avatar acts as a proxy for a user's profile picture or entity in a website.</p>
+        <div class="avatar-container mb">
+            <img src="https://picsum.photos/200" alt="avatar-img-1" class="avatar-img sml">
+            <img src="https://picsum.photos/300" alt="avatar-img-3" class="avatar-img mdm">
+            <img src="https://picsum.photos/400" alt="avatar-img-3" class="avatar-img lrg">
+            <img src="https://picsum.photos/500" alt="avatar-img-3" class="avatar-img x-lrg">
```

Comment:
You can use class name for different size avatars as `avatar-sm`, `avatar-md`, `avatar-lg`, `avatar-xl` instead of given size classes i.e. `lrg`

Reply:
class name changed, thank you for pointing out the errors!